src/plugins/transitionHooks/common.types.ts

```diff
@@ -4,6 +4,7 @@ export interface Route {
         [key: string]: any;
     };
     hostname: string;
+    route: string;
```

Comment:
Isn't it a breaking change adding required property to a public interface?

Reply:
In this case no, as this object is an input to the plugin.